lib/screens/tabs.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:delivery_app/mappers/screen_title_mapper.dart';
+import 'package:delivery_app/mappers/selected_screen_mapper.dart';
+import 'package:delivery_app/widgets/customized_search_bar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class TabsScreen extends StatefulWidget {
+  const TabsScreen({super.key});
+
+  @override
+  State<StatefulWidget> createState() {
+    return _TabsScreenState();
+  }
+}
+
+class _TabsScreenState extends State<TabsScreen> {
+  int _selectedPageIndex = 0;
+
+  void _selectPage(int index) {
+    setState(() {
+      _selectedPageIndex = index;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color.fromARGB(255, 239, 239, 239),
+      appBar: PreferredSize(
+        preferredSize: const Size.fromHeight(200),
+        child: AppBar(
+          flexibleSpace: Stack(
```

Comment:
Add the same background of `Scaffold`
```suggestion
        child: AppBar(
        backgroundColor: const Color.fromARGB(255, 239, 239, 239),
          flexibleSpace: Stack(
```